apps/epl_ets/src/epl_ets_viz_map.erl

```diff
@@ -0,0 +1,21 @@
+%%%-------------------------------------------------------------------
```

Comment:
Let's keep all functionality related to Vizceral maps in apps/epl/src/epl_viz_map.erl as other modules/applications can reuse it in the future.